src/pepr/operator/crd/sources/exemption/v1alpha1.ts

```diff
@@ -0,0 +1,99 @@
+import { V1CustomResourceDefinitionVersion, V1JSONSchemaProps } from "@kubernetes/client-node";
+
+export const v1alpha1: V1CustomResourceDefinitionVersion = {
+  name: "v1alpha1",
+  served: true,
+  storage: true,
+  additionalPrinterColumns: [
+    {
+      name: "Status",
+      type: "string",
+      description: "The status of the exemption",
+      jsonPath: ".status.phase",
+    },
+    {
+      name: "Age",
+      type: "date",
+      description: "The age of the exemption",
+      jsonPath: ".metadata.creationTimestamp",
+    },
+  ],
+  subresources: {
+    status: {},
+  },
+  schema: {
+    openAPIV3Schema: {
+      type: "object",
+      properties: {
+        status: {
+          type: "object",
+          properties: {
+            observedGeneration: {
+              type: "integer",
+            },
+            phase: {
+              enum: ["Pending", "Ready", "Failed"],
+              type: "string",
+            },
+          },
+        } as V1JSONSchemaProps,
+        spec: {
+          type: "object",
+          properties: {
+            exemptions: {
+              type: "array",
+              description: "Policy exemptions",
+              items: {
+                type: "object",
+                required: ["policies", "matcher"],
+                properties: {
+                  description: {
+                    type: "string",
+                    description:
+                      "A description of this exemption, this will become part of the exemption name",
+                  },
+                  policies: {
+                    description: "A list of policies to override",
+                    type: "array",
+                    items: {
+                      type: "string",
+                      enum: [
+                        "Disallow_Host_Namespaces",
```

Comment:
this will have a small maintenance tail I think, but like seeing this as an enum as the type system will give us TS validation and the API server will give us free validation on the runtime side.